spec/models/complaint_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe FakeData::Complaint do
+  let(:complaint) { FakeData::Complaint.new }
+
+  describe "#data" do
+    it "returns the data for a single complaint" do
+      data = complaint.data
+
+      expect(data[:id]).to_not be_nil
+      expect(data[:id]).to be_a(String)
+      expect(data[:attributes]).to be_a(Hash)
+      expect(data.dig(:relationships, :grantAward, :meta, :id)).to be_a(String)
+      expect(data[:links]).to be_a(Hash)
+    end
+  end
+
+  describe "#datetime_string" do
+    it "returns a string formatted as ISO-8601 UTC datetime" do
```

Comment:
[nitpick (nonblocking)] this test and the `#identifier` one are duplicating tests found in `ffaker_warpper_spec` so I don't think they add a lot of value here.

Reply:
oh yes i forgot to remove! thank you.